### DISCUSSION TITLE
who can solve these problems?

### DISCUSSION THREAD

**Discussion by creative2113** [Finalgoal231/finalGoal#41] [Discuss about technical stack] [answered]
I think this project is very poor.
 - SEO problem
 - loading speed problem
 - backend validation and frontend validation problem
 - style problem
 - I wanna add the loading page
I am interesting the developer who can solve these problems.

**Comment by noahstoner** [comment, marked as answer]
I will solve this issue:

backend validation and frontend validation problem

**Reply by creative2113** [reply]
how can solve this problem?

**Reply by noahstoner** [reply]
Hi, @creative2113 
Please break this discussion topic down into every issues.
Create your issues!!!

**Reply by noahstoner** [reply]
Then I will close this discussion

**Reply by creative2113** [reply]
yes

**Comment by creative2113** [comment]
I am looking for a developer can handle this problem.

**Comment by noahstoner** [comment]
Hi, @creative2113
Please break this discussion topic down into every issues.
Create your issues!!!



Then I will close this discussion

**Reply by creative2113** [reply]
what is the matter?

**Reply by creative2113** [reply]
I see.
